vcpkg-init/mint-standalone-bundle.ps1

```diff
@@ -82,6 +84,8 @@ try {
         Pop-Location
     }
 
+    Set-Content -Path "$Destination\vcpkg-version.txt" -Value $VcpkgBaseVersion -NoNewLine -Encoding Ascii
+    Copy-Item -Path "$PSScriptRoot/vcpkg-cmd.cmd" -Destination 'out/vcpkg-cmd.cmd'

```

Comment:
Why does this come from `$PSScriptRoot` and not `$SignedFilesRoot` like `vcpkg-init.cmd`?

Reply:
>Why does this come from $PSScriptRoot and not $SignedFilesRoot like vcpkg-init.cmd?

`vcpkg-init.cmd` is a copy of the same file as `vcpkg-init.ps1`. As such it is actually a powershell file and is thus signable. `vcpkg-cmd.cmd` is not also a powershell file and as a result to my understanding it isn't signable.

Reply:
(`VsDevCmd.bat` doesn't have a signature either)